recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 
 requirements:
   build:
-    - {{ compiler('go') }} 1.13
+    - {{ compiler('go') }} 1.17
```

Comment:
Looks like `go` 1.17 isn't available yet:
https://github.com/conda-forge/go-feedstock/pull/136

Reply:
Yeah hoped we had done that one already.  Lemme see if i can get that up and going.

Reply:
It was on my TODO to look at updating the patch there but I'm spread pretty thin... as I'm sure you are too! 😬

Reply:
https://github.com/conda-forge/go-activation-feedstock/pull/4 should get a 1.17 up and ready